addins/Cake.Scripting.Transport.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.Scripting.Transport
```

Comment:
Should be filtered as it is part of Cake and not an addin

Reply:
Done. Cake.Scripting.Transport added to blacklist